inginious/common/filesystems/local.py

```diff
@@ -142,7 +142,10 @@ def _recursive_overwrite(self, src, dest):
                 self._recursive_overwrite(os.path.join(src, file),
                                           os.path.join(dest, file))
         else:
-            shutil.copyfile(src, dest, follow_symlinks=False)
+            dirname = os.path.dirname(dest)
```

Comment:
A good way to import someone's changes is to use ``cherry-pick``. This avoids stealing the authorship of @nrybowski in this case :) I'll merge his PR first, so you can remove this from the PR.